.circleci/config.yml

```diff
@@ -79,7 +78,7 @@ jobs:
 
       - run: |
           set -euo pipefail
-          kurtosis run ./kurtosis/main.star --enclave ci-check "{\"kurtosis_package_indexer_version\": \"dev\", \"github_user_token\": \"${KURTOSISBOT_GITHUB_TOKEN}\", \"is_running_in_ci\": \"${CI}\"}"
```

Comment:
this changes CI, to use locally built image on CI runs - this will reflect latest changes of the service on the branch.